portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,14 +38,25 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the request
+    String maxCommentsString = request.getParameter("max-comments");
+
+    // Convert the input to an int or Integer.MAX_VALUE
+    int maxComments;
+    try {
+      maxComments = Integer.parseInt(maxCommentsString);
+    } catch (NumberFormatException e) {
```

Comment:
Might want to move catch to the next line? 
Seems like a coding standard, https://docs.microsoft.com/en-us/dotnet/standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.
try {
}
catch {
}

Reply:
I followed the Google coding standard example [here](https://google.github.io/styleguide/javaguide.html) (4.1.2 section). Not sure which is better

Reply:
Ah, didn't notice Google had one for Java. 

We can continue with yours.